docs/UserGuide.adoc

```diff
@@ -840,6 +968,12 @@ These commands can only be used in Statistics Mode.
 [width="90%",cols="^25%,^15%,30%,30%",options="header",]
 |=======================================================================
 ^|Command ^|Shortcut ^|Usage ^|Example
-a|<<Revenue, `revenue`>> a|`*r*` a|`revenue [d/DAY] [m/MONTH] [y/YEAR]` a|`revenue d/1 m/1 y/2019` +
+a|<<Yearly, `yearly`>> a|`*y*` a|`yearly` a|
+Displays up to 30 revenue from recent years in the Revenue Panel
```

Comment:
awkward phrasing, consider: "Displays the revenue from up to the past 30 years in the Revenue Panel"